cs/v1/CS_v1.0.0.yaml

```diff
@@ -61,17 +61,18 @@ paths:
                 example: 1.0.0
               description: |
                 SemVer used to indicate the version of the contract (API version) returned
-            Link:
+            Next-Page-Cursor:
               schema:
                 type: string
-                example : <https://api.dcsa.org/v1/point-to-point-routes?limit=20&cursor=f8dcb4c4-3fba-41ef-9991-2be97dad57dc>; rel="next",<https://api.dcsa.org/v1/point-to-point-routes?limit=20&cursor=f8dcb4c4-3fba-41ef-9991-2be97dad57dc>; rel="prev"
+                maxLength: 1024
+                example : fE9mZnNldHw9MTAmbGltaXQ9MTA=
```

Comment:
Is this now correct? I would assume that an example should include the url (or relative / example url)? @gj0dcsa do you have any opinion on this?